MoltenVK/MoltenVK/Commands/MVKCommandEncoderState.h

```diff
@@ -355,11 +355,11 @@ class MVKResourcesCommandEncoderState : public MVKCommandEncoderState {
 						   MVKArrayRef<uint32_t> dynamicOffsets,
 						   uint32_t& dynamicOffsetIndex);
 
-	/** Encodes the Metal resource to the Metal command encoder. */
-	virtual void encodeArgumentBufferResourceUsage(MVKShaderStage stage,
-												   id<MTLResource> mtlResource,
-												   MTLResourceUsage mtlUsage,
-												   MTLRenderStages mtlStages) = 0;
+	/** Encodes the indirect use of the Metal resource to the Metal command encoder. */
```

Comment:
Note that this must *also* be done for resources with `MTLHazardTrackingModeUntracked`, although we don't use that... [yet](https://github.com/KhronosGroup/MoltenVK/issues/1597) 😉 .

Reply:
Yes. #1597 would be good. Thanks for the reminder. I've moved that higher on my list of ToDos. 

It's too bad GitHub doesn't have an issue priority system to keep things high on the list. I guess we could fake it with labels. P1, P2, etc.